src/e2e-test/features/bigtable/BigTableToBigTable.feature

```diff
@@ -0,0 +1,126 @@
+# Copyright © 2024 Cask Data, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License. You may obtain a copy of
+# the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations under
+# the License.
+@BigTable @BIGTABLE_SOURCE_TEST
+Feature: BigTable source - Verification of BigTable to BigTable Successful Data Transfer
+
+  @BIGTABLE_SINK_TEST
+  Scenario: To verify data is getting transferred from BigTable source table to BigTable sink table
```

Comment:
How is this test different from 3rd test i.e. "To verify data is getting transferred from BigTable source table to NON existing BigTable sink" ?

Reply:
the difference between 1 and 3 is in first we are validating that source table is here by using validate plugin step line but in 3 we are not validating that and we are simply going as non existing tab.